tests/test_charge_rate.py

```diff
@@ -0,0 +1,58 @@
+import os, sys
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
+
+from battery_management_system.measurement_units import *
+from battery_management_system.charge_rate import ChargeRate
+from exception_handlers import get_error_type
+
+
+def test_low_charge_rate():
+    assert get_error_type(ChargeRate(Celcius(0.1)).is_low) == NotImplementedError

```

Comment:
Charge rate is one of the battery parameter and temperature is one too. It is not clear why celcius object is used in testing charge rate.

Reply:
@prithamvshetty understood, this is a mistake